.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Bug report
+about: Report a bug report to help us improve the tool.
+title: ''
+labels: 'bug'
```

Comment:
I'm not sure what is different between `bug` and  `cmty:bug-report`.
Does "cmty" mean the word "Community"?
Which should I use?

Reply:
cmty was a bug reporting platform that is no longer in use